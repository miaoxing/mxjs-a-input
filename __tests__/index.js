import Input from '../';
import {render} from '@testing-library/react';

describe('Input', () => {
  test('basic', () => {
    const result = render(<Input/>);

    expect(result.container).toMatchSnapshot();
  });
});
