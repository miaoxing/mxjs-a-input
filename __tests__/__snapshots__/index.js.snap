// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input basic 1`] = `
<div>
  <span
    class="ant-input-affix-wrapper"
  >
    <input
      class="ant-input"
      type="text"
      value=""
    />
    <span
      class="ant-input-suffix"
    />
  </span>
</div>
`;
